#request
bool set_linear_rail
---
#response
bool success
